src/components/timeline/TrackCounterGraph.js

```diff
@@ -99,20 +112,25 @@ class TrackMemoryCanvas extends React.PureComponent<CanvasProps> {
       // Draw the chart.
       const rangeLength = rangeEnd - rangeStart;
       ctx.lineWidth = deviceLineWidth;
-      ctx.strokeStyle = ORANGE_50;
-      ctx.fillStyle = '#ff940088'; // Orange 50 with transparency.
+      ctx.strokeStyle = graphStrokeColor;
+      ctx.fillStyle = graphFillColor; // Blue 50 or Orange 50 with transparency.
```

Comment:
I would remove this comment, since it will be easy to get out of date, and was only relevant with the hex value.